### ISSUE TITLE
[QUESTION] AttributeError: An error occurred during model prediction: 'dict' object has no attribute 'scores'

### ISSUE THREAD

**Issue by Evstrife** [Unbabel/COMET#184]
#### What is your question?
An error occurred during model prediction: 'dict' object has no attribute 'scores'
<img width="1067" alt="Screenshot 2023-11-19 at 7 47 08 PM" src="https://github.com/Unbabel/COMET/assets/42874576/2f4c4540-7c0f-4c6e-87bd-252e41233476">
<img width="648" alt="Screenshot 2023-11-19 at 7 47 14 PM" src="https://github.com/Unbabel/COMET/assets/42874576/6d70e827-fd89-4d29-8cb6-1ae3ad6307f1">

#### Code
```
echo -e "10 到 15 分钟可以送到吗\nPode ser entregue dentro de 10 a 15 minutos?" >> src.txt
echo -e "Can I receive my food in 10 to 15 minutes?\nCan it be delivered in 10 to 15 minutes?" >> hyp1.txt
echo -e "Can it be delivered within 10 to 15 minutes?\nCan you send it for 10 to 15 minutes?" >> hyp2.txt
echo -e "Can it be delivered between 10 to 15 minutes?\nCan it be delivered between 10 to 15 minutes?" >> ref.txt
comet-score -s src.txt -t hyp1.txt -r ref.txt
```

#### What have you tried?
I tried unbabel-comet== 2.0.1, which previously worked well with my env.
Also, tried unbabel-comet==2.1.0 and 2.2.0, both encountered same Attribute Error

#### What's your environment?

 - OS: Linux (AWS SageMaker)
 - Packaging 
pip = 23.3.1
 - Version 
unbabel-comet = 2.0.1/2.1.0/2.2.0
pytorch = 1.13
python = 3.9.0

**Comment by xiaoxin83121** [comment]
Same bug here

**Comment by xiaoxin83121** [comment]
You can simply replace the code `pred.scores` in the original python file in site-package to `pred["scores"]` for quick fix, that works for me.

**Comment by vrublack** [comment]
For me, installation from source worked (as opposed to the PyPi package).

**Comment by Evstrife** [comment]
> You can simply replace the code `pred.scores` in the original python file in site-package to `pred["scores"]` for quick fix, that works for me.

Thanks! Editing site package works for me.
Hopefully we can get an fix with PyPi and huggingface soon

**Comment by ymoslem** [comment]
Hi @Evstrife This was fixed by this merged pull request #185 

For now, you can reinstall COMET using this command:
```
pip3 install git+https://github.com/Unbabel/COMET.git
```
